AGENTS.md

```
@@ -74,9 +65,62 @@ All components in `packages/ui`:
-5. **Type safety** - avoid `any` unless necessary
-6. **Don't run dev servers** in automation
-7. **Search narrowly** - avoid reading large files/assets
+3. **Type safety** - avoid `any` unless necessary
+4. **Don't run dev servers** in automation
+5. **Search narrowly** - avoid reading large files/assets
+
+## Project Structure
+
+All projects in this repo should be structured like this:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)